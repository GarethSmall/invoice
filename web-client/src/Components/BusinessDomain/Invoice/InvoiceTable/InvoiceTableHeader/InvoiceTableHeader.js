// @flow
import React, { PureComponent } from 'react';

type Props = {};
export default class InvoiceTableHeader extends PureComponent<Props> {
  render() {
    return (
      <thead>
        <tr>
          <th>Item</th>
          <th>Qty</th>
          <th>Price</th>
          <th>Total</th>
          <th>Edit</th>
          <th>Delete</th>
        </tr>
      </thead>
    );
  }
}