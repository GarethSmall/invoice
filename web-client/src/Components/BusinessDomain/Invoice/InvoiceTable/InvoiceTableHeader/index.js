//@flow
import InvoiceTableHeader from './InvoiceTableHeader';
export default InvoiceTableHeader;
