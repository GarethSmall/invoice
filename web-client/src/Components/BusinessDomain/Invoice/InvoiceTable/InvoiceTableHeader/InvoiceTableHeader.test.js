// @flow
import React from 'react';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';
import shallow from 'enzyme/shallow';
import InvoiceTableHeader from './';
const { describe, test, expect } = global;

Enzyme.configure({ adapter: new Adapter() });

describe('InvoiceTableHeader', () => {
  test('should render', () => {
    const component = shallow((
      <InvoiceTableHeader />
    ));
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
