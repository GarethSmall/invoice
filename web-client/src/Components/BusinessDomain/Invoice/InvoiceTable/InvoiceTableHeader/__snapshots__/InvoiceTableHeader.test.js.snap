// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InvoiceTableHeader should render 1`] = `
<thead>
  <tr>
    <th>
      Item
    </th>
    <th>
      Qty
    </th>
    <th>
      Price
    </th>
    <th>
      Total
    </th>
    <th>
      Edit
    </th>
    <th>
      Delete
    </th>
  </tr>
</thead>
`;
